plugins/@grouparoo/customerio/src/migrations/000001-plugin-customerio-syncMode.ts

```diff
@@ -0,0 +1,31 @@
+export default {
+  up: async function (migration) {
```

Comment:
We should also migrate the existing `apiKey` option to `trackingApiKey` here (and do the inverse in the `down` section)